litex/tools/litex_json2dts_zephyr.py

```diff
@@ -137,10 +166,28 @@ def generate_dts_config(csr):
         if name not in overlay_handlers.keys():
             print('No overlay handler for:', name, 'at', hex(value))
 
+    cnf += ' -DCONFIG_LITEX_CSR_DATA_WIDTH={}'.format(
+        csr['constants']['config_csr_data_width'],
+    )
+
     return dts, cnf
 
 
 # helpers
+def get_registers_of(name, csr):
+    registers = csr['csr_registers']
+    data_width = csr['constants']['config_csr_data_width']
```

Comment:
Is this used anywhere?

Reply:
Yes, in updated handlers above. If you want, I can move it up, so it is interpreted, before functions that use it.

Reply:
Moved function just below imports and removed unused `data_width` variable